ex3.js

```diff
@@ -0,0 +1,50 @@
+// Scrieți o funcție pentru a găsi numărul de apariții al unei litere într-un șir
+
+// Funcția letterCount() returnează numărul apariții al unei litere trimise ca parametru dintr-un șir dat.
+
+// Exemplu 1:
+
+// Input: sir = “Îmi place programarea” litera = “a”
+
+// Output: 4
+
+//Model 1
+
+function letterCount(str) {
+
+    str = str.toLowerCase();
+    const letter = "a";
+    let count = 0;
+
+    for (let i = 0; i < str.length; i++) {
+        const el = str[i];
+
+        if (letter.includes(el)) {
```

Comment:
```suggestion
        if (el === letter) {
```
fiind ambele o litera, imi pare ca e mai optim asa